_query-dsl/compound/hybrid.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Hybrid
+parent: Compound queries
+grand_parent: Query DSL
+nav_order: 70
+---
+
+# Hybrid query
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/neural-search/issues/244).    
+{: .warning}
```

Comment:
This is not an experimental feature now. We have fixed the flag.

Reply:
Feature is enabled by default as Navneet mentioned, it's possible to disable it using a feature flag ```plugins.neural_search.hybrid_search_disabled``` (corresponding PR  https://github.com/opensearch-project/neural-search/pull/274)